README.md

```diff
@@ -2,45 +2,43 @@
 
 [![Build Status](https://travis-ci.org/mapbox/patrol-rules-github.svg?branch=master)](https://travis-ci.org/mapbox/patrol-rules-github)
 
-A set of rules implemented using [lambda-cfn](https://github.com/mapbox/lambda-cfn) and designed to run on a [Patrol](https://github.com/mapbox/patrol) stack.  The rules in this repository all aim to monitor an organizations Github repositories and users for best practices, security and compliance. Read more about the Patrol architecture on the [Patrol project](https://github.com/mapbox/patrol).
+A set of functions implemented using [lambda-cfn](https://github.com/mapbox/lambda-cfn) to monitor an organizations Github repositories and users for best practices, security and compliance. Part of the Mapbox [Patrol](https://github.com/mapbox/patrol) security framework.
 
-### Usage
+### Deploying
 
-Follow the steps on the [Patrol](https://github.com/mapbox/patrol) readme to set up your own Patrol stack on AWS which makes use of the patrol-rules-github rules.  Follow instructions on Patrol on how to enable or disable particular rules and how to deploy on your own AWS account.
+Please see the [lambda-cfn README](https://github.com/mapbox/lambda-cfn)
```

Comment:
I see your point about maintaining multiple sets of related documentation. We can have someone test out the lambda-cfn docs and see if there's anything missing that's Patrol-specific that we can either add here or to the lambda-cfn README.